### ISSUE TITLE
Empty value of SSML.ph

### ISSUE THREAD

**Issue by rdeltour** [w3c/epubcheck#139]
_From [jtchen0901@gmail.com](https://code.google.com/u/jtchen0901@gmail.com/) on November 15, 2011 11:42:22_

PROBLEM: Given an empty value of SSML.ph, Epubcheck3.0b3 reports no errors or warnings. METHOD: 1. Download the ssml.xhtml file of http://code.google.com/p/epubcheck/source/browse/branches/epub3/com.adobe.epubcheck.test/testdocs/30/single/xhtml/valid/ssml.xhtml?r=174 2. Clear the value of the first ssml:ph attribute, i.e.
 from
 <p ssml:ph="[pra'h-fessah]">Prof.</p>
 to 
 <p ssml:ph="">Prof.</p>
3. Pack the ssml.xhtml file into ssml.epub, and run the command.
java -jar epubcheck-3.0b3\epubcheck-3.0b3.jar ssml.epub RESULT: The epubcheck reports no warning or error.
Basically, an empty pronunciation makes no sense when the ssml.ph has been specified. EXPECTED: Some warning/error message should be shown, just like the invalid error/warning message caused by an empty SSML.alphabet value. PLATFORM: Epubcheck 3.0b3, WinXP SP3 ADDITIONAL INFO: none

_Original issue: http://code.google.com/p/epubcheck/issues/detail?id=139_
